### PR TITLE
Ai implementation final

### PR DESCRIPTION
**Pull request by Greg-Owen**
Implemented AI chatbot environment, adding working API.

### CODE REVIEW

File: client/builder.html [hunk 1]
```diff
@@ -161,7 +161,7 @@ <h1>
     
     <script src="./js/builder.js"></script>
     <link href="https://cdn.jsdelivr.net/npm/bootstrap@5.3.2/dist/css/bootstrap.min.css" rel="stylesheet" integrity="sha384-T3c6CoIi6uLrA9TneNEoa7RxnatzjcDSCmG1MXxSR1GAsXEV/Dwwykc2MPK8M2HN" crossorigin="anonymous">
-    <script src="js/auth.js"></script>
+    <!-- <script src="js/auth.js"></script> -->
```
Comment (0Thomas1) [inline client/builder.html]: delete this line

**Review by 0Thomas1 — COMMENTED** [review]
LGTM just delete that commented line please

**Comment by Scansy** [comment]
> LGTM just delete that commented line please

Yeah what he said

**Review by Scansy — APPROVED** [review]
LGTM!

**Pull request merged**